ts-nestjs/src/express/main.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * 
+ * @klotho::execution_unit {
+ *   id = "microsrv-api"
+ * }
+ */
+
+
+import express = require('express')
+import { addUser, getUsers } from './users';
+
+const expressApp = express()
+const router = express.Router();
+router.use(express.json())
+
+router.get('/users', async (req, res) => {
+  try {
+    const users = await getUsers()
+    res.send(users);
+  } catch (e) {
+    console.log("error getting users", e)
+    res.status(500).send(e)
+  }
+});
+
+router.put('/users/:user', async (req, res) => {
+  const user = req.params.user
+  try {
+    await addUser(user)
+    res.status(201).send(`Created ${user}`)
+  } catch (e) {
+    console.log("error adding user", e)
+    res.status(500).send(e)
+  }
+})
+
+expressApp.use(router)
+
+/* @klotho::expose {
+ *  target = "public"
+ *  id = "expressApp"
+ * }
+ */
+  expressApp.listen(3000, async () => {
```

Comment:
It's not clear to me what this module is for. This sets up a listener on the same port as the nestJs app.